symfony/twig-bundle/6.4/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Symfony\\Bundle\\TwigBundle\\TwigBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "templates/": "templates/"
+    },
+    "conflict": {
+        "symfony/framework-bundle": "<5.3"
```

Comment:
Is it still necessary? The bundle has a [conflict rule](https://github.com/symfony/symfony/blob/v6.4.0-RC1/src/Symfony/Bundle/TwigBundle/composer.json#L41) with `symfony/framework-bundle <5.4`